### DISCUSSION TITLE
Which model can be used with the Completions API in Azure OpenAI

### DISCUSSION THREAD

**Discussion by larkwins** [larkwins/langchain-examples#5] [Q&A] [answered]
I'm using Azure OpenAI, which model can be used with the Completions API? Like **text-davinci-003** in OPENAI.

**Comment by larkwins** [comment, marked as answer]
GPT-3.5 Turbo is used with the Chat Completion API. GPT-3.5 Turbo (0301) can also be used with the Completions API.

https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#gpt-35-models

---

**Model Name: GPT-3.5 Turbo
Model Version: 0301**


![image](https://github.com/luckpunk/langchain-examples/assets/3422759/f0b1c9e0-4d65-4c60-ae88-1b55e1e8f147)


```python
from langchain.llms import AzureOpenAI

API_TYPE = "azure"
BASE_URL = "https://xxx.openai.azure.com/"
API_KEY = "yyy"
DEPLOYMENT_NAME = "gpt-35-turbo-0301"
API_VERSION = "2023-09-15-preview"

azure_llm = AzureOpenAI(
    openai_api_base=BASE_URL,
    openai_api_version=API_VERSION,
    deployment_name=DEPLOYMENT_NAME,
    openai_api_key=API_KEY,
    openai_api_type=API_TYPE,    
)

azure_llm("Today is Tuesday, so yesterday is")
```

**Comment by LazaUK** [comment]
Hello @luckpunk, you may consider 2 models:
- **gpt-35-turbo v0301**, as you already discovered above, or
- **gpt-35-turbo-instruct**

https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#gpt-35
